cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
> Where would that happen? In the .env file?

That's how I set it in [this experiment](
https://github.com/cfpb/cfgov-refresh/pull/2244/commits/59a00604e4a00c32d4200e8ccd3e581c9eb10b37#diff-eb951766c3e0a8d12740b6236ded5a9fR145). If there's a more elegant way to set the defaults that may make sense to do it there. It seems counter to what environment variables are—essentially constants—to reference them but when they're not set have (potentially different) fallback values set throughout the codebase. The original reference should be what gets the fallback value at the time it is set.